torch/csrc/jit/script/compiler.h

```diff
@@ -177,6 +177,22 @@ struct TORCH_API AnnotateValue : public SugaredValue {
   }
 };
 
+// matched against for special handling of getattr expressions
+struct TORCH_API GetAttrValue : SugaredValue {
```

Comment:
This is never used outside of compiler.cpp why did it get exposed?

Reply:
Yes these are actually never used out of compiler, I just want to group all those sugaredValue definitions in a single place to make easy references since there're too many places for different ones. If compiler.h is meant to specifically only contains the definitions that will used outside, then I can move them back to compiler.